src/transform.ts

```diff
@@ -133,22 +144,26 @@ export interface WindowTransform {
   window: WindowFieldDef[];
 
   /**
-   * A two element specification about how large the sliding window is. The first element indicates the start and the second element indicates the end. If `null` is specified for the start, it will include everything before the current point. If `null` is specified for the end, it will include everything after the endpoint. For example a frame of `[-5,5]` says the window should include 5 previous objects and 5 after objects. The default is `[null, 0]`, which means include everything in the window. `[null, null]` would mean include everything in the window.
+   * A frame specification as a two-element array indicating how the sliding window should proceed. The array entries should either be a number indicating the offset from the current data object, or null to indicate unbounded rows preceding or following the current data object. The default value is `[null, 0]`, indicating that the sliding window includes the current object and all preceding objects. The value `[-5, 5]` indicates that the window should include five objects preceding and five objects following the current object. Finally, `[null, null]` indicates that the window frame should always include all data objects.
```

Comment:
>  The default value is `[null, 0]`, indicating 

For example, `[null, 0]` indicates that ...

(No need to repeat "default" twice -- If you haven't do `yarn run site` to check the page, please do as read through the docs as if you're a user and see if everything make sense.)